libraries/AP_OSD/AP_OSD_Screen.cpp

```diff
@@ -1200,6 +1213,7 @@ void AP_OSD_Screen::draw_batused(uint8_t x, uint8_t y)
 //Thanks to night-ghost for the approach.
 void AP_OSD_Screen::draw_message(uint8_t x, uint8_t y)
 {
+    x = 2; //messages get rendered across a whole row. User param x is ignored.
```

Comment:
what's the impact of this? wouldn't we need an option for it? or is it already the case that `x` was being ignored earlier?

Reply:
This is a behavior change, and I believe that it is a good one. I don't think it makes sense to allow users to set x to a high value, since English text is always written left to right. This line change, along with the few line changes lower down in this method, force all text to be centered on the screen. See the last 5 seconds of the youtube video. I can also dig up some more footage, if requested. I think it looks pretty cool :)

Reply:
sounds legit to me. maybe then in long-term it would make sense to change the signature of this function and its usages - remove the `x` param (?)

what  if some piece(s) of code would use this to show multiple short messages on the same line (e.g. coming from different modules/functions), but that would be weird (as the message is sometimes scrollable)...

Reply:
This should be in a separate PR. Its a unrelated change in behavior.